ui/apps/platform/src/messages/common.ts

```diff
@@ -81,6 +81,9 @@ export const resourceLabels = Object.freeze({
     SECRET: 'secret',
     CONTROL: 'control',
     CVE: 'CVE',
+    IMAGE_CVE: 'Image CVE',
+    NODE_CVE: 'Node CVE',
+    CLUSTER_CVE: 'Platform CVE',
```

Comment:
Backend called it "Cluster..." throughout their code, and then decided that "Platform" was a better name to convey to the meaning to users.

I'm trying to keep the internal representation consistent with the API ("cluster"), while change the text displayed.

Reply:
I see, that makes sense why you did it that way then. @md2119 since the backend moved towards calling it `Platform`, would it make sense to accommodate that terminology in the APIs? If not, maybe you can elaborate on it. The downside with this is the back-and-forth between calling it `Clusters` and `Platform`, which may be confusing

Reply:
I believe there is no instance in the frontend or the backend where we identify vulns as `Cluster CVEs`. We have direct classes—k8s, istio, and openshift. We are introducing this class for the first time. Therefore, I do not think there should be a worry about backward compatibility.